src/Shell/Task/TemplateTask.php

```diff
@@ -331,7 +334,7 @@ public function bakeActions(array $actions, $vars)
     /**
      * handle creation of baking a custom action view file
      *
-     * @return void
+     * @return void|bool
```

Comment:
Invalid. It can never be Void when returning sth else. Then it is Null. Also the other lines.